lib/proofer/vendor/mock.rb

```diff
@@ -22,18 +22,18 @@ def submit_answers(question_set, session_id = nil)
 
       def submit_financials(financials, session_id = nil)
         if financials.is_a?(Hash) && financials.values.first == '12345678'
```

Comment:
I think this is what most people type in when they're manually testing, at least I do. Perhaps we can make the invalid value something less common? Maybe `09876543` or `01234567`?